### ISSUE TITLE
Improve text box color contrast for GovDelivery subscribe

### ISSUE THREAD

**Issue by chriskuangGSA** [GSA/digitalcorps.gsa.gov#10]
Any chance we can change the input text color for the GovDelivery subscribe box at the bottom of each page? Right now, it's black on dark purple when you go to type your email, which is hard to read (might not pass contrast/accessibility checks).
